### PR TITLE
Percentage values are now recalculated correctly after excluding sites.

### PR DESCRIPTION
**Pull request by jasonrsadler**
Core implementation: https://github.com/brave/brave-core/pull/752

### CODE REVIEW

File: src/bat_publishers.cc [hunk 1]
```diff
@@ -525,8 +538,12 @@ void BatPublishers::synopsisNormalizerInternal(ledger::PublisherInfoList* newLis
   }
   size_t currentValue = 0;
   for (size_t i = 0; i < list.size(); i++) {
-    list[i].percent = percents[currentValue];
-    list[i].weight = weights[currentValue];
+    // if (list[i].excluded != ledger::PUBLISHER_EXCLUDE::EXCLUDED) {
```
Comment (NejcZdovc) [inline src/bat_publishers.cc]: please remove comments

File: src/bat_publishers.cc [hunk 2]
```diff
@@ -481,23 +490,27 @@ void BatPublishers::synopsisNormalizerInternal(ledger::PublisherInfoList* newLis
...
-    realPercents.push_back((double)list[i].score / (double)totalScores * 100.0);
-    percents.push_back((unsigned int)std::lround(realPercents[realPercents.size() - 1]));
-    double roundoff = percents[percents.size() - 1] - realPercents[realPercents.size() - 1];
-    if (roundoff < 0.0) {
-      roundoff *= -1.0;
-    }
-    roundoffs.push_back(roundoff);
-    totalPercents += percents[percents.size() - 1];
-    weights.push_back((double)list[i].score / (double)list.size() * 100.0);
+    // if (list[i].excluded != ledger::PUBLISHER_EXCLUDE::EXCLUDED) {
```
Comment (NejcZdovc) [inline src/bat_publishers.cc]: please remove comments

File: src/bat_publishers.cc [hunk 3]
```diff
@@ -481,23 +490,27 @@ void BatPublishers::synopsisNormalizerInternal(ledger::PublisherInfoList* newLis
   }
   double totalScores = 0.0;
   for (size_t i = 0; i < list.size(); i++) {
-    totalScores += list[i].score;
+    // if (list[i].excluded != ledger::PUBLISHER_EXCLUDE::EXCLUDED) {
```
Comment (NejcZdovc) [inline src/bat_publishers.cc]: please remove comments

File: src/bat_publishers.cc (hunk 1 above)
Comment (jasonrsadler) [reply src/bat_publishers.cc]: Done.

File: src/bat_publishers.cc (hunk 2 above)
Comment (jasonrsadler) [reply src/bat_publishers.cc]: Done.

File: src/bat_publishers.cc (hunk 3 above)
Comment (jasonrsadler) [reply src/bat_publishers.cc]: Done.

File: src/bat_publishers.cc [hunk 4]
```diff
@@ -527,6 +536,9 @@ void BatPublishers::synopsisNormalizerInternal(ledger::PublisherInfoList* newLis
   for (size_t i = 0; i < list.size(); i++) {
     list[i].percent = percents[currentValue];
     list[i].weight = weights[currentValue];
+    // } else {
```
Comment (NejcZdovc) [inline src/bat_publishers.cc]: please remove this comment

Comment (jasonrsadler) [reply src/bat_publishers.cc]: Done.

**Pull request merged**